WordPress/wordpress-playground
# Rrefactor `defineWpConfigConsts` to work on virtual file system

## What?



Remove `defineVirtualWpConfigConsts` in favor of `defineWpConfigConsts`, adding a `virtualize` option.

## Why?



Code optimization discussed here https://github.com/WordPress/wordpress-playground/pull/343#discussion_r1195701721

## Testing Instructions

1. Observe the blueprint tests pass
2. Run `nx preview wp-now start --path=/path/to/wordpress-plugin-or-theme`
3. Observe there is no `playground-consts.json` file added to the root of wordpress.
4. Observe the wp-config.php was not modified.
